app/components/work_package_relations_tab/index_component.rb

```diff
@@ -11,13 +11,14 @@ class WorkPackageRelationsTab::IndexComponent < ApplicationComponent
 
   attr_reader :work_package, :relations, :children, :directionally_aware_grouped_relations
 
-  def initialize(work_package:, relations:, children:)
+  def initialize(work_package:, relations:, children:, scroll_target_id: nil)
```

Comment:
Might by a personal preference, but imho `scroll_target_id` is not a good name. The scroll target could also be the container around that is scrolled and it is not clear what ID exactly is referenced here. I'd suggest something like `scroll_to_id`